### PR TITLE
🧹 remove source-file-secure

### PR DESCRIPTION
**Pull request by pedroslopez**
## What

As of https://github.com/airbytehq/airbyte/pull/30984, source-file-secure has been integrated into source-file, and this is no longer needed.

Bye!

## How

Delete code

## 🚨 User Impact 🚨

Users should not see any impact with this change since everyone has been moved over to source-file.

### CODE REVIEW

**Comment by pedroslopez** [comment]
Current dependencies on/for this PR:
* master
  * **PR #31477** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/31477" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/31477?utm_source=stack-comment).

File: airbyte-integrations/connectors/source-file/README.md [hunk 1]
```diff
@@ -149,7 +149,6 @@ All of your dependencies should go in `setup.py`, NOT `requirements.txt`. The re
 You've checked out the repo, implemented a million dollar feature, and you're ready to share your changes with the world. Now what?
 1. Make sure your changes are passing unit and integration tests
 2. Bump the connector version in `Dockerfile` -- just increment the value of the `LABEL io.airbyte.version` appropriately (we use SemVer).
-3. In addition to bumping the connector version of `source-file`, you must also increment the version of `source-file-secure` which depends on this source. The versions of these connectors should always remain in sync. Depending on the changes to `source-file`, you may also need to make changes to `source-file-secure` to retain compatibility.
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/source-file/README.md:152]: Very happy to remove this line

**Review by evantahler — APPROVED** [review]
Screenshot 2023-10-16 at 1 55 57 PM
NICE.

**Comment by octavia-approvington** [comment]
It's time <br/> fine lets go

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**